docs/source/maintenance/permissions.rst

```diff
@@ -31,6 +31,9 @@ List of Permissions
     * - Permission Name
       - Category
       - Type
+    * - `root`_ 
+      - All
```

Comment:
Please apply the change above and the change blow to the `matrix.csv`.
Otherwise, resulting headings look unclear.
<img width="744" alt="image" src="https://user-images.githubusercontent.com/36847043/63267192-2e5b5380-c29a-11e9-91b6-182705e04fdb.png">

After the fixes it is going to look like:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/36847043/63267320-711d2b80-c29a-11e9-9f5e-9c36698574fb.png">

Reply:
Done